frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -125,99 +145,45 @@ const DatasetRow: FC<Props> = ({
             checked={selected === dataset.id}
           />
           <div>{name}</div>
-          <ErrorTooltip isFailed={isFailed} error={error} />
+          {!isLoading && (
+            <ErrorTooltip isFailed={isFailed} error={error} type={type} />
+          )}
         </TitleContainer>
         {isLoading && (
           <UploadStatus
+            isConverting={
+              getConversionStatus(datasetStatus) ===
+              CONVERSION_STATUS.CONVERTING
+            }
             isValidating={
               datasetStatus.validation_status === VALIDATION_STATUS.VALIDATING
             }
             progress={datasetStatus.upload_progress}
-            cancelUpload={() => {
-              deleteDataset(dataset.id);
-            }}
+            datasetId={dataset.id}
+            collectionId={dataset.collection_id}
           />
         )}
       </DetailsCell>
-      <Popover values={tissue} isLoading={isLoading} isFailed={isFailed} />
-      <Popover values={assay} isLoading={isLoading} isFailed={isFailed} />
-      <Popover values={disease} isLoading={isLoading} isFailed={isFailed} />
-      <Popover values={organism} isLoading={isLoading} isFailed={isFailed} />
-      <CellCount cellCount={cell_count} isLoading={isLoading} />
+      <Popover values={tissue} isLoading={isMetadataLoading} />
```

Comment:
To keep thing DRY, consolidate `isFailed` and `isLoading` to just `isMetadataLoading`